lib/config/presets/index.ts

```diff
@@ -45,6 +45,9 @@ const gitPresetRegex = regEx(
   /^(?<repo>~?[\w\-. /]+)(?::(?<presetName>[\w\-.+/]+))?(?:#(?<tag>[\w\-./]+?))?$/
 );
 
+// shallow mode preset to ignore regex
+const whitesourcePresetRegex = regEx(/^.*?>whitesource\//);
```

Comment:
```suggestion
const whitesourcePresetRegex = regEx(/^.+?>whitesource\//);
```

there should be at least one char

```suggestion
const whitesourcePresetRegex = regEx(/^(?:.+?>)?whitesource\//);
```

So we filter `whitesource/merge-confidence:beta` too?

Reply:
Yes we are not including in the log